src/components/DateRangePicker/index.js

```diff
@@ -24,7 +34,7 @@ const DateRangePicker = ({
     }
   }, [init]);
```

Comment:
i'm not sure if init is still being used

Reply:
refactored